### PR TITLE
[Python API] add_outputs api

### PR DESCRIPTION
**Pull request by akuporos**
### Details:
 - Extend Function API -- add_outputs
 - Add Tensor descriptor

### Tickets:
 - 70965
 - 67432

### CODE REVIEW

**Review by AlexeyLebedev1 — APPROVED** [review]

**Comment by sdurawa** [comment]
Do we have tests for Tensor?

**Comment by akuporos** [comment]
> Do we have tests for Tensor?

Now yes, I've added them

File: runtime/bindings/python/src/pyopenvino/graph/node.cpp [hunk 1]
```diff
@@ -149,6 +149,23 @@ void regclass_graph_Node(py::module m) {
...
+                Returns the tensor for output i
+
+                Parameters
+                ----------
+                i : int
+                    Index of the output.
+
+                Returns
+                ----------
+                get_output_tensor : TensorDescriptor
```
Comment (jiwaszki) [inline runtime/bindings/python/src/pyopenvino/graph/node.cpp:166]:
```suggestion
                get_output_tensor : descriptor.Tensor
```

File: runtime/bindings/python/src/pyopenvino/graph/node_output.hpp [hunk 2]
```diff
@@ -81,4 +81,14 @@ void regclass_graph_Output(py::module m, std::string typestring)
...
                     Set of Inputs.
                )");
+    output.def("get_tensor",
+               &ov::Output<VT>::get_tensor,
+               py::return_value_policy::reference_internal,
+               R"(
+                A reference to the tensor descriptor for this output.
+                Returns
+                ----------
+                get_tensor : TensorDescriptor
```
Comment (jiwaszki) [inline runtime/bindings/python/src/pyopenvino/graph/node_output.hpp:91]:
```suggestion
                get_tensor : descriptor.Tensor
```

File: runtime/bindings/python/src/pyopenvino/graph/node.cpp [hunk 3]
```diff
@@ -149,6 +149,23 @@ void regclass_graph_Node(py::module m) {
...
+                    TensorDescriptor of the output i
```
Comment (jiwaszki) [inline runtime/bindings/python/src/pyopenvino/graph/node.cpp:167]: `Tensor of ...`

File: runtime/bindings/python/src/pyopenvino/graph/node_output.hpp [hunk 4]
```diff
@@ -81,4 +81,14 @@ void regclass_graph_Output(py::module m, std::string typestring)
...
+                    TensorDescriptor of the output.
```
Comment (jiwaszki) [inline runtime/bindings/python/src/pyopenvino/graph/node_output.hpp:92]: `Tensor of ...`

File: runtime/bindings/python/src/pyopenvino/graph/node.cpp (hunk 1 above)
Comment (akuporos) [reply runtime/bindings/python/src/pyopenvino/graph/node.cpp]: fixed

File: runtime/bindings/python/src/pyopenvino/graph/node_output.hpp (hunk 2 above)
Comment (akuporos) [reply runtime/bindings/python/src/pyopenvino/graph/node_output.hpp]: fixed

File: runtime/bindings/python/src/pyopenvino/graph/node.cpp (hunk 3 above)
Comment (akuporos) [reply runtime/bindings/python/src/pyopenvino/graph/node.cpp]: fixed

File: runtime/bindings/python/src/pyopenvino/graph/node_output.hpp (hunk 4 above)
Comment (akuporos) [reply runtime/bindings/python/src/pyopenvino/graph/node_output.hpp]: fixed

**Review by jiwaszki — APPROVED** [review]

**Pull request merged**